index.js

```diff
@@ -51,6 +51,11 @@ module.exports = function(source) {
 		config.plugins = config.plugins.concat(this.options[configKey].lessPlugins || []);
 	}
 
+	// Also support webpack 2 loader.options.
+	if (query.lessPlugins) {
```

Comment:
Could you name it `plugins`?

Could you refactor the config stuff to use `loaderUtils.getLoaderConfig(this, "lessLoader")`? This function handles the config stuff and merging.